source/val/validation_state.cpp

```diff
@@ -455,17 +455,7 @@ spv_result_t ValidationState_t::RegisterFunctionEnd() {
 
 Instruction* ValidationState_t::AddOrderedInstruction(
     const spv_parsed_instruction_t* inst) {
-  if (in_function_body()) {
```

Comment:
This is moved into the yucky bit above so that in_function_body() and in_block() are correctly defined by ModuleLayout.